### ISSUE TITLE
Can't set permissions for Atrium Folders feature

### ISSUE THREAD

**Issue by pban02** [nuvoleweb/atrium_folders#9]
Great module. 
For users with admin permissions everything is working fine. But authenticated users don't have any options to create folders. This is all in a brand new Atrium installation with no content. Any hints on how to get this working? 
TIA

**Comment by enricostano** [comment]
Hi,

to let authenticated users create new folders I've modified users permissions (/admin/user/permissions), in "features module > create folder content" and this worked for me.

bye,
enrico

**Comment by pban02** [comment]
Thanks - I tried to do that but the system doesn't let me change these permissions.
After clicking the appropriate permissions and saving nothing happens. The system is refusing to change the permissions and there is no message. 
Have I missed out on something basic here? Do I have to do something to allow permissions for new features to be set?

**Comment by enricostano** [comment]
I'm having the same behaviour with the "delete own/any folder contents" and "edit own/any folder contents" but "create folder content" works... it's strange..

**Comment by pban02** [comment]
Yes - in fact I can't change any permissions at all! Investigating further.
